storage/indexer.go

```diff
@@ -213,13 +209,13 @@ func (i *Indexer) Get(ctx context.Context, opts *packages.GetOptions) (packages.
 	return i.packageList, nil
 }
 
-func (i *Indexer) transformSearchIndexAllToPackages(sia searchIndexAll) (packages.Packages, error) {
+func (i *Indexer) transformSearchIndexAllToPackages(sia *searchIndexAll) packages.Packages {
 	var transformedPackages packages.Packages
 	for j := range sia.Packages {
 		m := sia.Packages[j].PackageManifest
```

Comment:
`PackageManifest` is a pointer. As this list comes from `loadSearchIndexAll`  function is supposed to not be nil.

Do you think it would  be needed to check whether or not that field is nil ?